dialog_flow_chatgpt_skill/README.md

```diff
@@ -0,0 +1,37 @@
+## Description
+
+Customer service bot built on `dff`. Uses telegram as an interface.
+
+This bot is designed to answer any type of user questions in a limited business domain (book shop).
+
+[DNNC](#) model from Salesforce is used for intent retrieval. Available at port 4999.
+[ChatGPT](#) is used for generative question answering.
+
+In order for the bot to work, override the stubs in the environment file: [.env](.env).
+
+## DNNC
+
+DNNC model is available at port 4999. 
+
+Users can interact with the container via '/respond' endpoint.
```

Comment:
Last time `user` is mentioned it means the user of a bot, so this leads to confusion. Should replace with
```suggestion
Service bot interacts with the container via '/respond' endpoint.
```